### PR TITLE
DOC: add status update to roadmap page.

### PR DESCRIPTION
**Pull request by rgommers**

### CODE REVIEW

**Comment by rgommers** [comment]
Ignore the failing CI here - the linkchecking on TravisCI is implemented incorrectly. Will fix that up in a separate PR.

**Comment by stonebig** [comment]
https://pypi.anaconda.org/carlkl/simple/mingwpy/ is unchanged since 10 months, looking at it via https://anaconda.org/carlkl/mingwpy/files, so point [6] is a little bit odd in step 2, at the moment.

**Comment by rgommers** [comment]
> https://pypi.anaconda.org/carlkl/simple/mingwpy/ is unchanged since 10 months, looking at it via https://anaconda.org/carlkl/mingwpy/files, so point [6] is a little bit odd in step 2

You're right, thanks for catching that @stonebig. I wrote this summary a week ago based on re-reading lots of email traffic. I think there that @carlkl was in the process of publishing new wheels, but I jumped the gun here.

@carlkl should I delete this part of the text for now, or is reality going to catch up with my summary soon?

File: roadmap.rst [hunk 1]
```diff
@@ -15,8 +15,67 @@ and the PSF in early Jan 2016):
 
    proposal_december2015
 
-Current status:
+Status update June '16
+----------------------
 
-- phase 1 of the proposal has been completed (March 2016)
```
Comment (techtonik) [inline roadmap.rst:20]: link to phase 1 would be appreciated

File: roadmap.rst [hunk 2]
```diff
@@ -15,8 +15,67 @@ and the PSF in early Jan 2016):
...
-- work on phase 2 is in progress, see the
-  `Milestone on Github <https://github.com/mingwpy/mingwpy/milestones/Phase%202>`_
+**Progress on milestones in the proposal**
+
+Phase 1 of the proposal has been finished.  The compiler toolchain has now been
+re-built by several people based on the public build scripts and documentation.
```
Comment (techtonik) [inline roadmap.rst:24]: links to `public build script` and `documentation` would be nice to see, as well as prooflinks for re-built versions (just my perfectionism)

File: roadmap.rst [hunk 3]
```diff
@@ -15,8 +15,67 @@ and the PSF in early Jan 2016):
...
+
+Phase 2 is about halfway to being finished:
+
+- The MingwPy GitHub project is set up [2]_ and mailing list [3]_ activity is
+  picking up.
+- Technical design documentation is gradually growing [4]_.
+- Numpy and Scipy wheels (both 32-bit and 64-bit) can be built with MingwPy
+  [6]_, albeit with a modified numpy.distutils.  Once the numpy.distutils
+  patches have been integrated in Numpy master this milestone is done.
```
Comment (techtonik) [inline roadmap.rst:33]: link to patches?

File: roadmap.rst [hunk 4]
```diff
@@ -15,8 +15,67 @@ and the PSF in early Jan 2016):
...
+- Only continuous integration work still has to start.
```
Comment (techtonik) [inline roadmap.rst:34]: ? link to task/problem decomposition

**Comment by carlkl** [comment]
I will add some changes to Ralfs complement (but not before wednesday this week)

**Comment by stonebig** [comment]
1 year now since the unforgettable mingwpy-0.1.0b3. mingwpy is the only piece missing to Python3.5 on windows today.

**Pull request merged**